core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -323,6 +323,12 @@ public Transaction() {
         vLockTime = LockTime.unset();
     }
 
+    /** @deprecated */
```

Comment:
This should specify the supported alternative.

Reply:
Well, actually there is no need to pass the _almost-deprecated_ `NetworkParameters`, so the `@deprecated` JavaDoc tag should probably suggest calling the no-args constructor. That may be deprecated/replaced in the future, but if we suggest the no-args constructor as an alternative we can at least remove this constructor in 0.18.

Reply:
I agree. I added this in a commit on top.